index.js

```diff
@@ -11,38 +11,21 @@ function exec(cmd) {
 try {
     const version = core.getInput('version');
     console.log(`Installing foundationdb ${version} (${os.platform()})!`);
-    let base_url = `https://www.foundationdb.org/downloads/${version}`;
+    let base_url = `https://github.com/apple/foundationdb/releases/download/${version}`;
     switch (os.platform()) {
         case 'linux': {
-            let client_url = `${base_url}/ubuntu/installers/foundationdb-clients_${version}-1_amd64.deb`;
-            exec(`curl -O ${client_url}`);
+            let client_url = `${base_url}/foundationdb-clients_${version}-1_amd64.deb`;
+            exec(`curl -L -O ${client_url}`);
             exec(`sudo dpkg -i foundationdb-clients_${version}-1_amd64.deb`);
 
-            let server_url = `${base_url}/ubuntu/installers/foundationdb-server_${version}-1_amd64.deb`;
-            exec(`curl -O ${server_url}`);
+            let server_url = `${base_url}/foundationdb-server_${version}-1_amd64.deb`;
+            exec(`curl -L -O ${server_url}`);
             exec(`sudo dpkg -i foundationdb-server_${version}-1_amd64.deb`);
             break;
         }
-        case 'win32': {
```

Comment:
I'm wondering if we should fail on that case :thinking: Users will migrate from Clikengo to this, and might expect Windows's support.

Reply:
Yes, makes sense. I guess we could have a default case which logs a message saying that the os is unsupported and throws an error, then that error would get caught by try-catch and the action would fail?